src/data/actions/grades.js

```diff
@@ -34,7 +34,10 @@ const startedFetchingGrades = () => ({ type: STARTED_FETCHING_GRADES });
 const finishedFetchingGrades = () => ({ type: FINISHED_FETCHING_GRADES });
 const errorFetchingGrades = () => ({ type: ERROR_FETCHING_GRADES });
 const errorFetchingGradeOverrideHistory = () => ({ type: ERROR_FETCHING_GRADE_OVERRIDE_HISTORY });
-const gotGrades = (grades, cohort, track, assignmentType, headings, prev, next, courseId) => ({
+const gotGrades = (
+  grades, cohort, track, assignmentType, headings, prev,
+  next, courseId, totalUsersCount, filteredUsersCount,
```

Comment:
We probably shouldn't be using positional parameters for all of these; you can switch to what is effectively named parameters by taking a single object parameter, which you destructure using these same names.